docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -68,18 +56,27 @@ module load CUDA/11.8.0
 
 ![!nvidia-smi output](images/CudaVersion.png)
 
-When your job has been created and your environment created and activated from the terminal (see above [instructions](../../workflow_solutions/using_anaconda.md#create-an-environment)), run the below command.
+1. Access the terminal following the steps [here](#installing-anaconda-environments-using-the-terminal).
+
+1. [Create](../../workflow_solutions/using_anaconda.md#create-an-environment) and [activate](../../workflow_solutions/using_anaconda.md#activate-an-environment) an environment as stated in these links.
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    When installing packages, modules and libraries into environments, remember to also install `ipykernel` using `conda install ipykernel`. This way your activated environment would appear in the list of kernels in your Jupyter Notebook. We have a [section](../../cheaha/open_ondemand/ood_jupyter.md#working-with-anaconda-environments) in our docs detailing how to switch conda environments.
+
+<!-- markdownlint-enable MD046 -->
+
+1. When your job has been created and your environment created and activated from the terminal (see above [instructions](../../workflow_solutions/using_anaconda.md#create-an-environment)), run the below command.
```

Comment:
Let's write "job has started" to set it apart from the "environment created...".